lib/clean.js

```diff
@@ -46,6 +46,17 @@ var CleanCSS = module.exports = function CleanCSS(options) {
   };
 };
 
+
+// for compatibility with optimize-css-assets-webpack-plugin
+CleanCSS.process = function (input, opts) {
+  return new Promise(function (resolve) {
+    var cleanCss = new CleanCSS(opts);
+    var output   = cleanCss.minify(input);
+    resolve({ css: output.styles });
```

Comment:
The webpack plugin only expects/uses the css property of returned object: https://github.com/NMFR/optimize-css-assets-webpack-plugin/blob/master/index.js#L78